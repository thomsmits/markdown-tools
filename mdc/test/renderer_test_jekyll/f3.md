# Chapter
## Slide 1.2

```java
int i = 7;
i++;
```
