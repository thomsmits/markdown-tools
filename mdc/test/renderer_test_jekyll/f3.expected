---
layout: page
parent: Section 3
title: Chapter
nav_order: 0
---
<h1 class="fw-700 text-purple-300">Chapter</h1>
## Slide 1.2
<div class="fw-300 fs-3"></div>
```java
int i = 7;
i++;
```


